pages/documentation/configuring_theme.md

```diff
@@ -29,6 +29,8 @@ matomo:
 # Matomo domain where Matomo is running
 matomo_id:
 # Integer indicating the Matomo Website ID
+matomo_cookies: false
```

Comment:
I personally would leave the default behavior to the current behavior, which is `matomo_cookies: true` or not having this setting at all. Please also update https://elixir-belgium.github.io/elixir-toolkit-theme/configuring_theme

Reply:
Makes sense!

I wasn't sure what would happen if existing sites start using this functionality without adding `matomo_cookies` to their `_config.yml`. I.e. what is the default value of an undefined variable? What I gathered from my Googlings was that an empty value would be truthy (I read somewhere that only `false` and `NIL` are falsy), so then when users update ETT without adding `matomo_cookies` it would indeed be evaluated as true and hence the `unless site.matomo_cookies` clause would not trigger.

But it's very well possible I am mistaken! In any case, I will change the default value.

Reply:
Applied suggestion in b05d71f.

Reply:
Or is `NIL` actually what a non-defined value is? In that case I now understand your second more explicit `if false` suggestion! :D

Reply:
Indeed, NIL is the 'value' when a variable is not defined, and so Falsy! https://shopify.github.io/liquid/basics/truthy-and-falsy/ . I tried it locally so my suggestion should work!